docker/cheqd_node/node-runner.sh

```diff
@@ -5,16 +5,33 @@
 set -euox pipefail
 
 CHEQD_ROOT_DIR="$HOME/.cheqdnode"
+CHEQD_COMMON_DIR="$HOME/.common"
 
 # Init node config directory
 if [ ! -d "${CHEQD_ROOT_DIR}/config" ]
 then
     echo "Node config not found. Initializing."
-    cheqd-noded init $NODE_MONIKER --home 
+    cheqd-noded init $NODE_MONIKER --home ${CHEQD_ROOT_DIR}
 else
     echo "Node config exists. Skipping initialization."
 fi
 
+if [ -e "${CHEQD_COMMON_DIR}/genesis.json" ]
+then
+    echo "Use genesis.json from ${CHEQD_COMMON_DIR} directory"
+    GENESIS=$(cat ${CHEQD_COMMON_DIR}/genesis.json)
+else
+    echo "There is no genesis.json in ${CHEQD_COMMON_DIR} directory. Variable value GENESIS will be used"
+fi 
+
+if [ -e "${CHEQD_COMMON_DIR}/seeds.txt" ]
+then
+    echo "Use seeds.txt from ${CHEQD_COMMON_DIR} directory"
+    P2P_SEEDS=$(cat ${CHEQD_COMMON_DIR}/seeds.txt)
+else
+    echo "There is no seeds.txt in ${CHEQD_COMMON_DIR} directory. Variable value P2P_SEEDS will be used"
+fi
```

Comment:
Not sure where this is set/passed in. Maybe I'm just tired

Reply:
It's passed by mounting another volume in the `docker-compose`, especially `- ../../../persistent_chains/mainnet/:/cheqd/.common`